src/screens/Home.js

```diff
@@ -1,31 +1,40 @@
 import { render } from '../shared/dom.js';
 import ApiProvider from '../providers/ApiProvider.js';
-import { MovieCarousel } from '../components';
-import ConsoleLogger from '../utils/ConsoleLogger.js';
+import { Button, MovieCarousel } from '../components';
+import './Home.scss';
+import { redirect } from '../index.js';
 
 const template = async (movies) => `
-    <div>
-        <div id="home-screen">This is home screen</div>
+    <div id="homeScreen" class="homeScreen">
         <template id="home-movie-tiles"></template>
-        ${await MovieCarousel({ movies: movies[0], id: 'Carousel0' })}
-        ${await MovieCarousel({ movies: movies[1], id: 'Carousel1' })}
-        ${await MovieCarousel({ movies: movies[2], id: 'Carousel2' })}
-        ${await MovieCarousel({ movies: movies[3], id: 'Carousel3' })}
+        ${await MovieCarousel({
+            movies: movies[0],
```

Comment:
Operating on array is pointless here, in real world project we would probably get an array from API, and we could display these lists dynamically, and we would get the title as well

Here we could do that, but then we will have to mutate our data and there is actually no point of doing that, so just pass 3 separate arguments instead of array, would be much more readable